components/UiKit/Typography.tsx

```diff
@@ -0,0 +1,35 @@
+import styled from "styled-components";
+
+export const Heading3 = styled.h3`
+    font-size: 1.375rem;
```

Comment:
Lets use pixels for font-size everywhere, its easier to use figma numbers, and we don't need a calculator.